Dockerfile

```diff
@@ -32,6 +32,7 @@ LABEL name="Red Hat Messagign QE - Java CLI Image" \
 
 USER root
 
+RUN microdnf install util-linux -y
```

Comment:
Comment! Why do we install util-linux?

Probably does not matter too much, but a fancy way of running microdnf is (as shown e.g. https://github.com/skupperproject/skupper-router/blob/main/Containerfile#L22)
```suggestion
# install fallocate for use by tests
RUN microdnf -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install util-linux && microdnf clean all -y
```